pywbem/_cim_obj.py

```diff
@@ -2052,6 +2052,7 @@ def from_wbem_uri(wbem_uri):
                 _format("WBEM URI has an invalid format for its keybindings: "
                         "{0!A}", keybindings_str))
 
+        kb_assigns = []
```

Comment:
While this does not hurt, the matter of fact is that the regexp in WBEM_URI_KEYBINDINGS_REGEXP has a mandatory first matching group. So if that group does not match, the "if m is None:" check a few lines above would have caused an exception to be raised, so if the code gets here it is guaranteed that the first group was there. I think coverity does not analyze the regexp, so it cannot come to that conclusion.

What is really not needed because of the guaranteed presence of the first group is the "if m.group(1):". With that present though in the old code, I can understand why coverity misses the initialization.

If we take the guaranteed first group into account, the code could become:

```
        assert m.group(1)
        kb_assigns = [m.group(1)]
```

However, the initialization is also fine, and I don't see a need to change the proposed solution.